client/src/main/java/jass/client/model/SplashModel.java

```diff
@@ -103,7 +103,7 @@ protected Void call() {
                 }
             });
             // Check whether we can already login
-            tasks.add(() -> {
+            /*tasks.add(() -> {
```

Comment:
```suggestion
            tasks.add(() -> {
```
Uncomment function again (was just for testing)